### PR TITLE
Add the CONTRIBUTION.md file

### LINKED ISSUE #3: Create a contribution guide
Create a CONTRIBUTING.md with the guidelines on how to contribute to this project. Also, reference this file in the README

### PR DESCRIPTION
**Pull request by rafamarts**
Close #3 

I've added the first step of contribution file. Let's start a discussion to update it. Please, suggest anything. Thank you!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Contributing to Poppins  
+
+Thank you for your support! All contribution are very important to us. The OpenSource wouldn't be where it is today without contributors like you who are willing to take the time to improve it for everyone else.
```
[the remark this answers is not in the document]
Comment (jairsjunior) [reply CONTRIBUTING.md:3]: So cool!! 
Another tip is if you can mark the issue related to this PR, everyone will know when access the issue that it has a pending PR and facilitate for us keep the issues list updated.

**Comment by jairsjunior** [comment]
So cool!!
Another tip is if you can mark the issue related to this PR, everyone will know when access the issue that it has a pending PR and facilitate for us keep the issues list updated.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rafamarts) [reply CONTRIBUTING.md:3]: Awesome!! Thank you!

**Comment by tiagostutz** [comment]
> So cool!!
> Another tip is if you can mark the issue related to this PR, everyone will know when access the issue that it has a pending PR and facilitate for us keep the issues list updated.

[Here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) is a guideline on how to do it.

Closes #8

**Pull request merged**